desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersView.java

```diff
@@ -299,6 +319,14 @@ protected void activate() {
 
     private void updateNumberOfOffers() {
         numItems.setText(Res.get("shared.numItemsLabel", sortedList.size()));
+        groupColumn.setVisible(ocoIsInUse());
```

Comment:
Why do that in a method called `updateNumberOfOffers`? It has nothing to do with updating the Number of offers. We need to take more care of code quality otherwise Bisq 1 code gets un-manageable....